### ISSUE TITLE
Getting Started - step-by-step is unclear (where it links out to the changelog)

### ISSUE THREAD

**Issue by pbwolf** [clojure/clojure-site#470]
The instructions <https://clojure.org/guides/getting_started> are hard to follow step-by-step.  I installed an outdated version by accident.

The first major section, "Clojure installer and CLI tools", says "see the changelog for version information".  I didn't recognize it as part of the essential step-by-step path, and I skipped it.  (After all, the link right before it, to "command line tools", is definitely a rabbit hole.)  But what I didn't know is, the very specific version number in the ensuing steps is NOT the latest.  In short, the changelog reference was too subtle for me.

In a Getting Started page, a very clear step-by-step is important as each fork in the road deepens the reader's worry of ending up at a not-well-understood  place.  Please make the changelog an active part of the instructions, if it was intended as such, or else move it to the end in a "how to update" paragraph?

**Comment by puredanger** [comment]
Have redone a lot of details on this page, considering this closed.
